examples/ratatui/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "psp-ratatui-example"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+psp = { path = "../../psp", features = ["embedded-graphics"] }
+embedded-graphics = { version = "0.8.1", features = ["fixed_point"]}
+
+# Ratatui 0.30.0 has no_std support
+ratatui = { version = "0.30.0-alpha.4", default-features = false }
```

I bumped ratatui-core in mousefood, you should bump ratatui accordingly.
```suggestion
ratatui = { version = "0.30.0-alpha.5", default-features = false }
```